GitHub.Collectors/Cache/PointCollectorTableEntity.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.CloudMine.Core.Collectors.Cache;
+using Microsoft.CloudMine.GitHub.Collectors.Model;
+
+namespace Microsoft.CloudMine.GitHub.Collectors.Cache
+{
+    public class PointCollectorTableEntity : TableEntityWithContext
```

Comment:
Not sure how this cache works, could you please explain? I have a fear that this might not be doing what you are looking for.

Is this attempting to cache a particular request / record so that if we are already going to process it, we would not again? If this is the case, would not keeping track of the exact URL to call (instead of org. repo, record type) easier? Also, when do we remove stuff? After all, you want to re-collect some URLs over time in case something has changed. (This might come later in the PR)